### PR TITLE
Vcdat2.0 sterling

### PR DESCRIPTION
**Pull request by sterlingbaldwin**
This cleans up a bunch of stuff and adds a variable select modal. Still trying to get the dimension sliders to work so those arent included.

### CODE REVIEW

**Comment by doutriaux1** [comment]
some conflicts, but will be reviewing in my very limited capacity.

**Comment by doutriaux1** [comment]
@downiec please review too

**Comment by doutriaux1** [comment]
also we should rename this repo to `jupyter-vcdat-ext` when bringing it back into cdat organization

File: src/components/AddEditRemoveNav.tsx [hunk 1]
```diff
@@ -1,43 +1,78 @@
...
+                        <NavLink href="#" onClick={this.props.removeAction} title={this.props.removeText ? this.props.removeText : ""}>
+                            <i className={`glyphicon glyphicon-remove-sign disabled`}></i>
+                        </NavLink>
+                    </NavItem>
                 </div>
-            </nav>
+            </Nav>
         )
     }
+    // render() {
```
Comment (doutriaux1) [inline src/components/AddEditRemoveNav.tsx]: Do we want to keep these commented out for now?

File: src/components/LeftSideBar.tsx [hunk 2]
```diff
@@ -1,42 +1,103 @@
...
+    // handle click on a variable by opening dimension select modal
+    handleVarClick(e: any) {
+        debugger;
+    }
+    handleSetupConsole(console: any) {
+        this.console = console;
+        let script = [
+            'import cdms2',
+            'import vcs',
+            'x = vcs.init()'];
```
Comment (doutriaux1) [inline src/components/LeftSideBar.tsx:69]: @downiec that is probably where you want to add your function to query variables and and graphic methods.

**Review by doutriaux1 — CHANGES_REQUESTED** [review]
Look ok but need to make sure conflicts are solved and it doesn't hurt @downiec stuff

**Review by downiec — APPROVED** [review]

File: src/components/AddEditRemoveNav.tsx (hunk 1 above)
Comment (sterlingbaldwin) [reply src/components/AddEditRemoveNav.tsx]: No, I should have removed that. Its the old non-typescript bootstrap nav elements that were conflicting with the variable edit modal css.

File: src/components/LeftSideBar.tsx (hunk 2 above)
Comment (sterlingbaldwin) [reply src/components/LeftSideBar.tsx:69]: This is the setup script that gets run once on startup.

**Comment by sterlingbaldwin** [comment]
I resolved all the apparent conflicts, Im going to merge and rebuild and test it to make sure its all working since I was just eye-balling the conflict resolution.

**Pull request merged**